### DISCUSSION TITLE
Is it possible to use an accelerometer for control?

### DISCUSSION THREAD

**Discussion by takhir-nafikov** [littlektframework/littlekt#233] [Q&A] [answered]
Instead of buttons, I want to set the movement through the phone's tilts

**Comment by LeHaine** [comment, marked as answer]
As of right now, there is no abstraction for it. But that doesn't mean you can't use it directly. If you are just using Android you could just read it using the Android API. But if you need it for KS too then you might need some sort of abstraction.

But that is a good idea for a feature.

**Reply by takhir-nafikov** [reply]
Thank you
